.gitignore

```diff
@@ -27,5 +27,4 @@ node_modules
 # Users Environment Variables
 .lock-wscript
 
-docs/
```

Comment:
I believe you want `doc` here to prevent checking in generated docs when working with a module in isolation. Reference: https://github.com/canjs/can-map/blob/master/.gitignore#L30